scenarios/aws/eks/run.go

```diff
@@ -243,6 +265,10 @@ func Run(ctx *pulumi.Context) error {
 			}
 		}
 
+		if err != nil {
+			return err
+		}
```

Comment:
Which `error` are we checking here ?
In case of non-`nil` `error`, shouldn’t have we returned earlier ?

Reply:
sorry i just forgot to remove this!